### ISSUE TITLE
Consider adding a feature where comments preceding the method get in the same fold

### ISSUE THREAD

**Issue by bruno-** [vim-utils/vim-ruby-fold#3]

**Comment by janko-m** [comment]
I would really like that latter option. I would like that I can configure vim-ruby-fold to fold only comments above methods. It would really help me to read Lotus' source code, because that code just has way too many comments.

**Comment by bruno-** [comment]
Hm.. my initial thought was to just fold all multi-line comments, no matter where their location is (above a class, above method etc). That ok?

**Comment by janko-m** [comment]
Yes, that would be great as well. There most often aren't any comments inside of code anyway (if it's written in a good way) and targeting all comments is probably simpler than just those above methods.
